_data/tool_and_resource_list.yml

```diff
@@ -1754,7 +1754,7 @@
   name: Research Data Management Platform (RDMP)
```

Comment:
I think the full name should be "Research Data Management Platform (RDMP) at Health Informatics Centre, University of Dundee"

Reply:
I agree with Bert, this is too long for the tool name. Even current name is too long 
![image](https://github.com/user-attachments/assets/0ddef92f-002c-464a-8688-4e808bf07886)
so I shortened it into RDMP and extended the description.

@bedroesb Bert can the tool id also be changed from  id: research-data-management-platform to id: rdmp? I do not know how to check if there will be no conflicts with such id (yaml file does not have such id yet but what about other places?).

Reply:
I can also help off course. btw I think with @bianchini88 we kinda decided to spell out full tool names with the acronym in parentheses.

Reply:
> I can also help off course. btw I think with @bianchini88 we kinda decided to spell out full tool names with the acronym in parentheses.

Do you mean that id should be `research-data-management-platform_(rdmp)`?

Reply:
@marikapop oh no, I ment the name, not the id. it is a good idea to keep the id short (with the acronym) So full support on that one!